content/blog/gatsby-blog/index.md

```diff
@@ -0,0 +1,323 @@
+---
+title: Create a blog with Gatsby.js
+date: "2019-12-23T22:12:03.284Z"
+author: franleplant
+description: "In this post we are going to cover the steps we took to create the nosleepjavascript.com blog 
+  in simple, easy, steps using the latest front end technologies: Gatsby.js (JAMstack), Nodejs, React,
+  Graphql, Javascript, Github Pages and Github Actions"
+todo:
+  - in the future it would be nice to add a section on to how to deploy to azure
```

Comment:
this is just metadata that we are not going to be rendering so it is effectively a "code comment"